### PR TITLE
fix: change username to displayName

### PR DESCRIPTION
**Pull request by janrembold**
This only fixes the "case 2" from https://virtual-identity.clickup.com/t/6656196/OB-7132
Will work on "case 1" and update when found but already created this MR so that we don't do that twice...

### CODE REVIEW

File: src/containers/bookings/components/NoBookings/noBookingsBooked.tsx [hunk 1]
```diff
@@ -60,8 +60,7 @@ export const NoBookingsBooked: React.FC<NoBookings> = ({ sessions }) => {
                             ${sessions
 								?.filter((session) => session.agency !== null)
 								?.map(
-									(consultant) =>
-										consultant.consultant.username
+									({ consultant }) => consultant.displayName
```
Comment (web-mi) [inline src/containers/bookings/components/NoBookings/noBookingsBooked.tsx]: 1. We don't have always filled up the displayName on all clients. Should be `consultant.displayName || consultant.userName`

2. This is coming from the core already. So please make another pr with the change inside the core.

**Review by web-mi — CHANGES_REQUESTED** [review]

File: src/containers/bookings/components/NoBookings/noBookingsBooked.tsx (hunk 1 above)
Comment (janrembold) [reply src/containers/bookings/components/NoBookings/noBookingsBooked.tsx]: Done

**Review by web-mi — APPROVED** [review]
